script.js

```diff
@@ -7,32 +7,38 @@ let books = [];
 function displayData() {
   let bookList = '';
   for (let i = 0; i < books.length; i += 1) {
-    bookList += `<div class=book>
-    <h3 class="title">${books[i].title}     
-      </h3>
-      <p class="author">
-      ${books[i].author}</p>
-      <div class="remove"><button onClick="deleteBook(${i})">Remove</button>
-      </div>
-      <hr>
-      <br>
+    bookList += `<div class="book">
+   <div>
+    <td>${books[i].title}     
+      </td>
+      <td> by </td>
+      <td>
+      ${books[i].author}</td></div>
+      <div>
+      <td><button onClick="deleteBook(${i})" class="remove">Remove</button>
+      </td></div>
      </div>
      `;
   }
   document.getElementById('tdBody').innerHTML = bookList;
 }
 
-function add() {
+class book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
```

Comment:
- [ ] Good job by using JavaScript `classes` as required for the project. So, I noticed that from the project's requirements all the methods have to be inside the classes. i.e.:
```javascript
class Book {
  constructor(title, author) {
    this.title = title;
    this.author = author;
  }

addBook() {
    code for the addBook method 
  }

  deleteBook() {
     code for the deleteBook method 
  }

 ....

 }
```